### PR TITLE
pfc_chayka.ini

### PR DESCRIPTION
**Pull request by den-88fish**
My first experience here on hosting configs. Tips and tricks are appreciated.

### CODE REVIEW

File: config/tracks/pfc_chayka.ini [hunk 1]
```diff
@@ -0,0 +1,735 @@
...
+SEMITRANSPARENT_SHADOWS = 1
+
+[SHADER_REPLACEMENT_...]
+MATERIALS=fence_?
+CAST_SHADOWS=1
+DOUBLE_FACE_SHADOW_BIASED=1
+
+[SHADER_REPLACEMENT_...]
+MATERIALS = gravel01_6_bake
+SHADER = ;ksGrass
```
[the remark this answers is not in the document]
Comment (den-88fish) [reply config/tracks/pfc_chayka.ini:32]: replacing the shader will help me get rid of the dark border in gravel

[the remark this answers is not in the document]
Comment (RustyChest) [reply config/tracks/pfc_chayka.ini:27]: Sure but once again, you're not actually doing anything?
The ```SHADER=``` parameter is to change the actual shader used, not the parameters
It is only to be specified when you're changing it from one shader to another

**Review by RustyChest — CHANGES_REQUESTED** [review]

File: config/tracks/pfc_chayka.ini [hunk 2]
```diff
@@ -0,0 +1,735 @@
...
+KEY_0 = ksEmissive
+VALUE_0 = 255,183,76, 0.02
+VALUE_0_OFF = 0, 0, 0
+KEY_1 = ksAlphaRef
+VALUE_1 = -193
+VALUE_1_OFF = 0, 0, 0
+CONDITION = NIGHT_SMOOTH
+ 
+[CustomEmissive]
+ACTIVE = 1
```
[the remark this answers is not in the document]
Comment (den-88fish) [reply config/tracks/pfc_chayka.ini]: > Active isn't needed
It turns out that it is necessary. After I removed this, the whole texture began to glow for me.when i leave ACTIVE = 1 then the areas that I have identified are illuminated

**Comment by RustyChest** [comment]
This PR is obsolete as there was an update to the config, now making it incompatible. Hope the pointers helped at the very least

**Pull request closed without merging**